GameEngine/visualizer.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, simulator: None, number_of_players: int = 2, field_size: tupl
         self._temp_initialized_data = False
         self._data_provided = False
         self._data_provided_buffer = ()
+        self.fps = 500
```

Comment:
fps = 500? This is maximum fps we allow the game - right? is it just happy number or you selected it for some reason? (probably we would be fine with much smaller value)

also, we probably should synchronise it with the `dt` (delta time) in the game simulator